tests/test_new_meshing_workflow.py

```diff
@@ -1199,15 +1199,16 @@ def test_switch_between_workflows(new_meshing_session):
 
     # 'import_geometry' is a watertight workflow command which is not available now
     # since we have changed to fault-tolerant in the backend.
-    with pytest.raises(RuntimeError):
+    # It seems to throw either RuntimeError or ValueError randomly.
```

Comment:
Seems to coming from either server (`RuntimeError`) or client (`ValueError`) based on some timing. We can probably change the error type on the server-side to `ValueError` as that is more appropriate.